nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/pom.xml

```diff
@@ -246,6 +246,11 @@
                                         <include>moment/README.md</include>
                                         <include>moment/LICENSE</include>
 
+                                        <!-- carbon icons -->
+                                        <include>@carbon/icons/svg/**/*</include>
```

Comment:
It would be great if we did not need to add yet another dependency here. Maybe we can use one of the available icons from font awesome or the custom flowfont created for this project?

Reply:
Good point. #af43e5c integrates carbon icon directly into the flowfont.